node-operators/validators/onboarding/register-in-symbiotic.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Register in Symbiotic
+description: Learn how to register your node in the Symbiotic registry as a first step to participate in the protocol, securing Tanssi-powered networks and earning rewards.
+icon: octicons-shield-check-24
+---
+
+# Register in Symbiotic
+
+## Introduction {: #introduction }
+
+--8<-- 'text/node-operators/validators/onboarding/run-a-validator/intro.md'
+
+Before being able to secure Tanssi-powered networks, node operators must [set up the node](/node-operators/validators/onboarding/run-a-validator/){target=\_blank} and register as operators in the Symbiotic protocol. This guide will walk you through the steps to successfully register your node as an operator.
```

Comment:
```suggestion
Before securing Tanssi-powered networks, node operators must [set up the node](/node-operators/validators/onboarding/run-a-validator/){target=\_blank} and register as operators in the Symbiotic protocol. This guide will walk you through the steps to successfully register your node as an operator.
```

Same feedback ehre as on the other page. What are we doing in this guide specifically? Anything else needed beforehand should be added to prereqs